packages/core/src/components/Timeline/styles.ts

```diff
@@ -0,0 +1,24 @@
+import { StyleSheet } from '../../hooks/useStyles';
+
+export const styleSheetItem: StyleSheet = ({ color, ui, unit }) => ({
+  wrapper: {
+    marginLeft: unit * 0.5 - 1,
+    borderLeft: ui.borderThick,
+    paddingLeft: unit * 2.5,
+    paddingBottom: unit * 7,
+  },
+
+  wrapper_secondary: {
+    paddingBottom: 0,
+    height: unit * 2.75,
```

Comment:
can this be minHeight? or not at all - use paddingBottom to make up the height difference or something? what if the translation is really long and the line wraps, or the line wraps in general? how does this height interact?

Reply:
@stefhatcher so for secondary items, we don't render the `children` prop in the "content" area of the timeline item - it gets rendered as the timeline item "title" - for the "secondary item" scenario - the wrapper will just be an _empty_ `<div />` - and we've provided this height in order to give it a minimum spacing between items - the screenshot should show the empty space I'm talking about  
![image](https://user-images.githubusercontent.com/839082/74888020-9d8d2f80-5331-11ea-9632-e9999b6326f0.png)

I set the `children` prop for a `secondary` item here to wrap, it looks OK to me - LMK if I've misunderstood

Reply:
FWIW it _could_ be `min-height` if thats preferred

Reply:
@Tgeo i hope you dont mind, but i was very intrigued by the secondary wrapping problem (wanted to preserve the spacing around the dot, while allowing the secondary content to wrap with the bar growing) and re-wrote this to use css grid 

https://github.com/airbnb/lunar/pull/322/commits/089344d899c8d7c20eeeb8e4b3b9fa35eeb29950

and pushed it up. thoughts?

in this screenshot the top is the new styles with wrapping secondary, with the previous below to compare (i also fixed the off by one pix in the screenshot, so ignore that part):
<img width="328" alt="Screen Shot 2020-02-20 at 7 56 29 AM" src="https://user-images.githubusercontent.com/306275/74956301-c0175b00-53ba-11ea-981a-5936cb2bba3b.png">

Reply:
@stefhatcher don't mind at all - I wish we did that more often